frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -22,11 +22,11 @@ const MonitoringDashboard: React.FC<MonitoringDashboardProps> = ({ match }) => {
         <title>Dashboard</title>
       </Helmet>
       <Grid className="co-m-pane__body" gutter="md">
-        <GridItem span={4} rowSpan={1}>
-          <MonitoringDasboardCountBlock />
+        <GridItem span={3} rowSpan={1}>
+          <MonitoringDasboardPodCount namespace={namespace} />
         </GridItem>
-        {_.map(queries, (q) => (
-          <GridItem span={4} key={q.title}>
+        {_.map(queries, (q, i) => (
+          <GridItem span={i === 0 ? 5 : 4} key={q.title}>
```

Comment:
any reason for this condition here?

Reply:
Yes, as per UX podCount width is smaller than other block and the first graph block is bigger than others.